alembic/versions/872f79201961_initial_migration.py

```diff
@@ -0,0 +1,49 @@
+"""initial migration
+
+Revision ID: 872f79201961
+Revises: 
+Create Date: 2024-11-01 01:18:04.110740
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '872f79201961'
+down_revision: Union[str, None] = None
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
```

Comment:
The type hint for `depends_on` is not necessary unless you have a specific reason to enforce type checking here. Alembic will handle this as a string, sequence of strings, or None.